src/main/java/io/cdap/plugin/google/common/APIRequestRetryer.java

```diff
@@ -51,8 +51,8 @@ public abstract class APIRequestRetryer {
   protected static final int LIMIT_RATE_EXCEEDED_CODE = 403;
   protected static final int BACKEND_ERROR_CODE = 500;
   protected static final int SERVICE_UNAVAILABLE_CODE = 503;
-  private static final int MAX_RETRY_WAIT = 200;
-  private static final int MAX_RETRY_COUNT = 8;
+  private static final int MAX_RETRY_WAIT = 300;
```

Comment:
nit: please use `time unit` in the constant `name`, for example `MAX_RETRY_WAIT_SECONDS` if it represents `seconds`.